python/cudf/cudf/tests/test_series.py

```diff
@@ -1614,6 +1614,37 @@ def test_axes(data):
         assert_eq(e, a)
 
 
+def test_series_truncate():
+    csr = cudf.Series([1, 2, 3, 4])
+    psr = csr.to_pandas()
+
+    assert_eq(csr.truncate(), psr.truncate())
+    assert_eq(csr.truncate(1, 2), psr.truncate(1, 2))
+    assert_eq(csr.truncate(before=1, after=2), psr.truncate(before=1, after=2))
+
+
+def test_series_truncate_invalid_axis():
+    csr = cudf.Series([1, 2, 3, 4])
+    msg = "No axis named 1 for object type Series"
+    with pytest.raises(ValueError, match=msg):
+        csr.truncate(axis=1)
+
+
+def test_series_truncate_datetimeindex():
+    dates = cudf.date_range("2021-01-01", "2021-01-02", freq="s")
```

Comment:
Could we reduce the data input size by only generating times between `2021-01-01 23:45:00` and `2021-01-01 23:46:00`? It seems excessive to test with 86,400 rows here when 60 rows will do.

Reply:
This comment was marked as resolved but it was not resolved.

Reply:
Done.